### PR TITLE
Add backend-br on others jobs repositories section

### PR DESCRIPTION
**Pull request by iagodahlem**
Add backend-br on others jobs repositories section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Confira abaixo as outras listas onde você poderá postar sua vaga:
 - [Vagas para desenvolvedores iOS e OSX](https://github.com/CocoaHeadsBrasil/vagas)
 - [Vagas pra desenvolvedores PHP](https://github.com/phpdevbr/vagas)
 - [Vagas pra desenvolvedores Stone Pagamentos](https://github.com/stone-pagamentos/vagas)
+- [Vagas para desenvolvedores(as) Back-end](https://github.com/backend-br/vagas)
```
Comment (wevtimoteo) [inline README.md]: Nice catch, WDYT to change all `desenvolvedores` to `desenvolvedorxs`? Or even add `es(as)` for all?

Comment (iagodahlem) [reply README.md]: Thanks :smile: 

Hmm, I think that's a good idea, more consistent and more inclusive for everyone.

Can I change it?

Comment (wevtimoteo) [reply README.md]: Yes, please @iagodahlem

**Review by wevtimoteo — APPROVED** [review]

**Comment by wevtimoteo** [comment]
Thank you @iagodahlem!

**Comment by iagodahlem** [comment]
No problem @wevtimoteo :smile:

**Pull request merged**